src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala

```diff
@@ -1,6 +1,6 @@
 package io.iohk.ethereum.crypto
 
-import io.iohk.ethereum.nodebuilder.SecureRandomBuilder
+import io.iohk.ethereum.security.SecureRandomBuilder
```

Comment:
(out of scope ?) I think `EcKeyGen` should be part of `security` package instead of `crypto`. I mean crypto package should contains only primitives.